.github/workflows/zap-vs-vulnerableapp.yml

```diff
@@ -0,0 +1,17 @@
+name: ZAP vs VulnerableApp
+
+on:
+  workflow_dispatch:
+
+jobs:
+  scan:
+    runs-on: [ubuntu-latest]
+    steps:
+    - name: Start vulnerableApp
+      run: docker run --rm -d -p 9090:8080 --name=owasp-vulnerableapp sasanlabs/owasp-vulnerableapp:latest
+    - name: Start ZAP
+      run: docker run --rm -d -p 8090:8090 --network host owasp/zap2docker-weekly zap-x.sh -daemon -host localhost -port 8090 -addonupdate -addoninstall domxss -config "api.addrs.addr.name=.*" -config api.addrs.addr.regex=true -config api.disablekey=true &
+    - name: Run ZAP API script
+      env:
+        GITHUB_TOKEN: ${{ secrets.ZAPBOT_TOKEN }}
+      run: docker run --rm --network host --env ZAPBOT_TOKEN=${{ secrets.ZAPBOT_TOKEN }} owasp/zap-testing ./zap-vs-vulnerableApp.sh -e 32 -n vulnerableApp-weekly-RBJ-M-M -t "Rel, Beta, DomXSS"
```

Comment:
@psiinon i have updated files but as docker owasp/zap-testing is not having the files so the action is failing. can you please help in this.

Reply:
Pull the image?

Reply:
Disregard for now, there are options but I’m camping and can’t dig in right now ⛺️